src/gluonts/model/deepstate/_estimator.py

```diff
@@ -154,6 +159,7 @@ def __init__(
         prediction_length: int,
         cardinality: List[int],
         add_trend: bool = False,
+        output_transform: Optional[ComposedBijectionHybridBlock] = None,
```

Comment:
As the normalizing flow bijections are inherently multivariate, for the Local-NKF, one would need to wrap these in a LocalBijection class which applies a local flow to each observations implicitly, and Jacobian term is handled appropriately (this has already been coded and should be added in this PR, I can share this with you)

Reply:
I would use the most general annotation for this argument, if possible: couldn't one use a generic `Bijection` here?